app/javascript/availability/components/aeon_link.jsx

```diff
@@ -18,12 +19,16 @@ const AeonLink = ({holding, locationText}) => {
         const itemID = encodeURIComponent(holding.itemID);
         const itemTypeID = holding.itemTypeID;
         const genre = itemTypeID === "ARCHIVES" ? "ARCHIVES" : "BOOK";
-        let aeonUrl = `https://aeon.libraries.psu.edu/Logon/?Action=10&Form=30` +
-            `&ReferenceNumber=${catkey}&Genre=${genre}&Location=${itemLocation}` +
-            `&ItemNumber=${itemID}&CallNumber=${callNumber}`;
+        let aeonUrl = "https://aeon.libraries.psu.edu/RemoteAuth/aeon.dll";
 
         $.get(`/catalog/${catkey}/raw.json`).then((data) => {
```

Comment:
What happens if this blows up with a 4/5xx?

Reply:
@awead - great point. I can move the code to set the url + clear the spinner into a finally block to make sure we do that stuff regardless of whether the AJAX call succeeds or fails.

```
$.get().then((data) => {
  // do stuff
}).finally(() => {
  setShowSpinner(false);
  setUrl(illUrl);
});
```

Reply:
@awead - for some reason putting this stuff in a `finally()` block didn't behave the way I expected it to, so I added the cleanup in a `catch()` block instead. I was getting some uncaught promise rejections in Jest and it was blowing up the RSpec availability tests. Should be good for re-review.

Reply:
Could we just use a plain 'ol javascript Promise and not bother with jQuery?

Reply:
Yeah, honestly I can't see a reason why we couldn't just use `fetch()`.

![fetch](https://media0.giphy.com/media/3otPoUjeyRisIDxPhK/giphy.gif?cid=6c09b952527c27d72ba092da8009a92c65aadad796e55c17&rid=giphy.gif&ct=g)

Reply:
How fetching!